### PR TITLE
Added log option to histogram operation

### PR DESCRIPTION
**Pull request by philippjfr**
This adds a log option to the histogram operation, which automatically generates logarithmic sampling for the histogram bins. Also includes a small fix to fix bokeh histograms with inverted axes.

### CODE REVIEW

File: holoviews/operation/element.py [hunk 1]
```diff
@@ -478,6 +478,9 @@ class histogram(ElementOperation):
     individually = param.Boolean(default=True, doc="""
       Specifies whether the histogram will be rescaled for each Raster in a UniformNdMapping.""")
 
+    log = param.Boolean(default=False, doc="""
+      Whether to apply log scaling """)
```
Comment (jlstevens) [inline holoviews/operation/element.py]: I would at least mention that it is log10....

Comment (philippjfr) [inline holoviews/operation/element.py]: True, that's what logspace uses, but it should still be explained.

**Comment by jlstevens** [comment]
Other than a small comment about the docstring, this PR looks fine and I am happy to merge.

**Comment by jlstevens** [comment]
Other than one unrelated test failure, the tests seem to be ok. Merging.

**Pull request merged**